include/singa/io/network/endpoint.h

```diff
@@ -0,0 +1,139 @@
+/************************************************************
+*
```

Comment:
would it be better to include all header files of network into a single include/singa/io/network.h file, and put the .cc files into src/io/network/ folder?